packages/wallets/src/components/WalletAddedSuccess/WalletAddedSuccess.tsx

```diff
@@ -22,10 +22,7 @@ const WalletAddedSuccess: React.FC<TWalletAddedSuccessProps> = ({
 }) => {
     const { isMobile } = useDevice();
     const description = 'Make a deposit into your new Wallet.';
-    const title = useMemo(
-        () => `Your ${currency} wallet (${landingCompany?.toUpperCase()}) is ready`,
-        [currency, landingCompany]
-    );
+    const title = useMemo(() => `Your ${currency} wallet is ready`, [currency]);
```

Comment:
v.minor/optional - arguably, the overhead of validating cache, checking dependencies array, memoising results is way bigger than cost of simple string concatenation :D :D